airflow/jobs.py

```diff
@@ -1434,7 +1434,7 @@ def _execute_helper(self, processor_manager):
                                                           State.NONE)
 
                 self._execute_task_instances(simple_dag_bag,
-                                             (State.SCHEDULED,))
```

Comment:
Please don't do this. You will double execute tasks as you can schedule them twice now.